frontend/public/module/k8s/k8s.ts

```diff
@@ -38,8 +39,40 @@ export const groupVersionFor = (apiVersion: string) => ({
   version: apiVersion.split('/').length === 2 ? apiVersion.split('/')[1] : apiVersion,
 });
 
+export const getLatestVersionForCRD = (crdVersions: CRDVersion[], legacyCRDVersion: string) => {
+  const served = crdVersions.filter((version) => version.served);
+  const sorted = served.sort((v1, v2) => {
```

Comment:
I'd make your sort function reusable. We can use it in other places like the Home -> Explore page.